modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -396,25 +405,25 @@ rtfctl get apps
 * Prints a list of all applications in a specific environment 
 +
 ----
-rtfctl get apps --namespace <environment>
+rtfctl get apps --app-namespace <appNamespace>
 ----
 
 * Displays an HTTP proxy configurations
 +
 ----
-rtfctl get http-proxy
+rtfctl get http-proxy --namespace <rtfNamespace>
```

Comment:
This is inconsistent. For parts of the docs, we don't say to use `--namespace <rtfNamespace>`, some parts we do. I think we should be consistent. Since you are already calling out at the top to use this flag in case of custom namespaces, I don't think we need to add it for every single command.